board/opentrons/ot2/rootfs-overlay/usr/bin/ot-blinkenlights

```diff
@@ -30,6 +30,7 @@ time.sleep(0.25)
 
 def handleSignal(signal_number, frame):
     gpio.set_button_light(blue=True)
+    os._exit()
```

Comment:
Per [the docs](https://docs.python.org/2/library/os.html#os._exit) this requires an exit status argument and indeed systemd shows the service stop having a big traceback.